### DISCUSSION TITLE
WinUI package is outdated

### DISCUSSION THREAD

**Discussion by malstraem** [Mapsui/Mapsui#1637] [Q&A]
I started some project with WinUI3 and used last mapsui packages from appveyor, but now appveyor's WinUI package is incompatible with last nuget's 4.0.0-beta.4 build.

When will it be updated?

**Comment by inforithmics** [comment]
In this pull request I reneabled the appveyor WinUI Build

https://github.com/Mapsui/Mapsui/pull/1630

and I think in the next beta uno and winui nuget packages can be released but 
@pauldendulk  decides this.

**Reply by pauldendulk** [reply]
Mapsui.WinUI 4.0.0-beta.5 was released. There is a getting started here but I did not test it recently.
https://mapsui.com/documentation/getting-started-winui.html
